addons/crm/tests/test_performances.py

```diff
@@ -92,13 +92,14 @@ def test_assign_perf_no_duplicates(self):
         leads.flush()
 
         with self.with_user('user_sales_manager'):
-            with self.assertQueryCount(user_sales_manager=589):  # 584-585 generally, sometimes 589
+            with self.assertQueryCount(user_sales_manager=589): # 584-585 generally, sometimes 589
```

Comment:
Linting: In commit "make TestLeadAssignPerf deterministic" you have some changes in test_performances.py that could be removed, as anyway you alter that file in the final commit "update query count".